server/tests/src/test/java/org/infinispan/server/functional/MemcachedOperations.java

```diff
@@ -28,4 +33,37 @@ public void testMemcachedOperations() {
       cache.set("k1", 0, "v1");
       assertEquals("v1", cache.get("k1"));
    }
+
+   @Test
```

Comment:
These changes seem fine, but do they belong in this commit? They aren't related to JDBC.